lib/components/flat_button_short.dart

```diff
@@ -0,0 +1,36 @@
+import 'package:flutter/material.dart';
+import 'package:seeds/design/app_colors.dart';
+import 'package:seeds/design/app_theme.dart';
+
+class FlatButtonShort extends StatelessWidget {
```

Comment:
is this a copy of the flat log button? 

Does it make sense to make the flag long button take a param 

1- full width
2 Wrap content

like an enum.